### PR TITLE
Updates pages for portfolio, transactions and stocks.

### PR DESCRIPTION
**Pull request by brandomoreno06**
### Story
As required for user stories number 6 and 7, pages for portfolio and transactions were updated. For css styling, I have installed boot-strap and popper-js via yarn. With the changes included on this PR, users can now see their portfolio and transactions. 

In relation to user story numbers 7 & 8(feature for buying and selling were already included in previous commits), UI has also been updated. Users will be able to see how may stocks they hold and all their transactions for the particular stock.

Also, the stocks page has been modified so he can see the list of stocks in detail.
The link for transactions, portfolio and stocks pages are placed on the header. 

### Screenshots
image
image
image
image

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
+  background-color: green !important;
+  border-color: green !important;
+  color: white !important;
+}
+
+.submit__transaction.sell {
+  background-color: rgb(156, 4, 4) !important;
+  border-color: rgb(156, 4, 4) !important;
+  color: white !important;
+}
```
Comment (analyn-cajocson) [inline app/assets/stylesheets/application.css]: add new line at the end of file

File: app/javascript/packs/new_transaction.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+selectTransaction.addEventListener('change', modifyTransactionType)
+selectTransaction.addEventListener('load', modifyTransactionType)
+
+selectTransaction.addEventListener('click', function () {
+  if (this.value == 'buy') {
+    this.style.borderColor = 'green'
+  } else {
+    this.style.borderColor = 'red'
+  }
+});
```
Comment (analyn-cajocson) [inline app/javascript/packs/new_transaction.js]: add new line at the end of file

File: app/javascript/stylesheet/application.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+@import "bootstrap";
```
Comment (analyn-cajocson) [inline app/javascript/stylesheet/application.scss]: add new line at the end of file

**Review by bryanarboledaa — APPROVED** [review]
This is good, approve.

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -1,19 +1,28 @@
...
+          <% else%>
+          <%= link_to "Sign up", new_user_registration_path %>
+          <%= link_to "Sign in", new_user_session_path %>
+          <%= link_to "Sign in as Admin", new_admin_session_path %>
+      <% end %>
+    </div>
   </div>
-<% end %>
\ No newline at end of file
+</div>
```
Comment (analyn-cajocson) [inline app/views/layouts/_header.html.erb]: add new line at the end of file

File: app/views/pages/home.html.erb [hunk 5]
```diff
@@ -1,30 +1,21 @@
...
+      <% @stocks.each do |stock| %>
+        <div>
+          <%= stock.code %>
+          <%= stock.current_price %>
+          <%= link_to 'Buy/Sell', new_stock_trade_path(stock) %>
+        </div>
+      <% end %>
+    </div>
+  <% end %>
+</div>
```
Comment (analyn-cajocson) [inline app/views/pages/home.html.erb]: add new line at the end of file

File: app/views/pages/landing.html.erb [hunk 6]
```diff
@@ -1 +1,3 @@
-<h1> Landing Page </h1>
\ No newline at end of file
+<div class="col mt-5">
+  <h1> Landing Page </h1>
+</div>
```
Comment (analyn-cajocson) [inline app/views/pages/landing.html.erb]: add new line at the end of file

File: app/views/pages/portfolio.html.erb [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+            <td class= "text-center"><%= stock.current_price %></td>
+            <td class= "text-center"><%= user_stock.quantity %></td>
+            <td class= "text-center"><%= user_stock.quantity * stock.current_price %></td>
+            <td class= "text-center"><%= link_to 'Buy/Sell', new_stock_trade_path(user_stock.stock_code) %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  <% end %>
+</div>
```
Comment (analyn-cajocson) [inline app/views/pages/portfolio.html.erb]: add new line at the end of file

File: app/views/pages/transactions.html.erb [hunk 8]
```diff
@@ -1,3 +1,3 @@
-<div>
+<div class= 'container mt-5'>
   <%= render 'shared/transactions', transactions: @transactions %>
 </div>
```
Comment (analyn-cajocson) [inline app/views/pages/transactions.html.erb]: add new line at the end of file

File: app/views/stocks/index.html.erb [hunk 9]
```diff
@@ -1,3 +1,24 @@
...
+        <tr>
+          <td><%= stock.code %></td>
+          <td><%= stock.name %></td>
+          <td class= "text-center"><%= stock.current_price %></td>
+          <td class= "text-center"><%= link_to 'buy/sell', new_stock_trade_path(stock) %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+</div>
```
Comment (analyn-cajocson) [inline app/views/stocks/index.html.erb]: add new line at the end of file

File: app/views/trades/new.html.erb [hunk 10]
```diff
@@ -1,37 +1,43 @@
...
-<% end %>
\ No newline at end of file
+  <% if current_user %>
+    <div class= "mt-5 mb-5 border-top">
+      <%= render 'shared/transactions', transactions: @transactions %>
+    </div>
+  <% end %>
+  
+  <%= javascript_pack_tag 'new_transaction' %>
+</div>
```
Comment (analyn-cajocson) [inline app/views/trades/new.html.erb]: add new line at the end of file

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (brandomoreno06) [reply app/assets/stylesheets/application.css]: Hi. Kindly enlighten me why there is a need to add a new line at the end of the file?

**Pull request merged**